### ISSUE TITLE
[Chore] Replace `lerna bootstrap` with `yarn`

### ISSUE THREAD

**Issue by alt1o** [DouyinFE/semi-design#1915]
The "bootstrap" command was removed by default in v7, and is no longer maintained.

[https://lerna.js.org/docs/legacy-package-management#replacing-your-usage-of-lerna-bootstraplerna-link](https://lerna.js.org/docs/legacy-package-management#replacing-your-usage-of-lerna-bootstraplerna-link)

**Comment by pointhalo** [comment]
Yes, we have paid attention to the issue you mentioned🤝 
After nx took over lerna, there were many changes in the v7 version. We did not lock the version before, but then we encountered some problems with dependency management and release-related CI, so we temporarily locked v6 and did not upgrade it.
From the perspective of long-term maintenance, we need to do this. Staying in a low version that stops service is not a good choice.But it won't be urgent for the time being. 
We will consider replacing it when we have some free time.

**Comment by alt1o** [comment]
Absolutely! I suggest posting an issue detailing the challenges you've encountered. This way, others might provide assistance and work together towards a solution.

**Comment by SudoUserReal** [comment]
感谢你的建议！确实如 @pointhalo 提到的，我们目前暂时锁定了 lerna 版本。

关于替换 `lerna bootstrap`，我们目前的情况是：
- 短期内暂时不会在当前版本做这个改动，因为涉及到的 CI 流程和依赖管理比较复杂，需要稳定运行
- 但这确实是我们规划中的事情，我们计划在下个 major 版本直接迁移到更原生的方案，可能是 npm workspaces 或者 bun，配合一些自建的脚本来管理 monorepo

这样既能解决 lerna v7 的兼容性问题，也能让构建流程更透明可控。到时候也会参考社区的一些最佳实践。

如果你有具体的迁移方案或者踩过坑，也欢迎继续分享，对我们后续规划会很有帮助 🙏
